README.md

```diff
@@ -1,2 +1,13 @@
 # InfiniteRecharge-2020
 Code for recharging a robot infinitely in the 2020 FRC game Infinite Recharge
+
+## Project setup
+After gradle has synced, either use the gradle menu to run
+> Tasks > help > addKtlintFormatGitPreCommitHook
+
+or in your commandline, run
+
+```
+chmod +x gradlew
+./gradlew addKtlintFormatGitPreCommitHook
```

Comment:
Is this in the terminal? Because it doesn't work when I run it there.

Reply:
Did you try running this with gradle, like the built in gradle thing in Intellij?

Reply:
I'm pretty sure this only works on linux. On windows just run 
`./gradlew adKtlintFormatGitPreCommitHook`

Reply:
If you are using the native windows terminal, omit the `./`